### PR TITLE
docs: update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by daphne-sfdc**
Updates to CONTRIBUTING.md as discussed during today's meeting.

[skip-validate-pr]

### CODE REVIEW

**Review by peternhale — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,16 +2,14 @@
 
 1. Familiarize yourself with the codebase by reading the [docs](docs), in
    particular the [development](contributing/developing.md) doc.
-1. Create a new issue before starting your project so that we can keep track of
-   what you are trying to add/fix. That way, we can also offer suggestions or
-   let you know if there is already an effort in progress.
+1. Create a new discussion before starting your project so that we can keep track of what you are trying to add/fix. If it fits in our roadmap, then we will review your contribution and work with you to get it merged. That way, we can also offer suggestions or let you know if there is already an effort in progress.
```
Comment (CristiCanizales) [inline CONTRIBUTING.md:5]: is the discussion to keep track or to validate/analyze if we will spend time on it?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,16 +2,14 @@
...
-1. Create a _topic_ branch in your fork based on the correct branch (usually the **develop** branch, see [Branches section](#branches) below). Note, this step is recommended but technically not required if contributing using a fork.
+1. Optional: Create a _topic_ branch in your fork based on the correct branch (usually the **develop** branch). Note, this step is recommended but technically not required if contributing using a fork.
 1. Edit the code in your fork.
-1. Sign CLA (see [CLA](#cla) below)
-1. Send us a pull request when you are done. We'll review your code, suggest any
-   needed changes, and merge it in.
+1. Sign the CLA (see [CLA](#cla) below)
+1. Send us a pull request when you are done. We'll review your code, suggest any needed changes, and merge it in.
+1. In your pull request, please provide us with a one-pager to help us understand your code. In the one-pager, explain what the change is, why it is necessary, and how you made the change. The one-pager should include a test plan so we can get an idea of how you have tested your code changes. We would also greatly appreciate a short demo video of the expected behavior of your feature.
```
Comment (CristiCanizales) [inline CONTRIBUTING.md:12]: should we mention here that the changes need unit tests as well before requesting a review?

File: CONTRIBUTING.md (hunk 1 above)
Comment (daphne-sfdc) [reply CONTRIBUTING.md:5]: Analyze if we will spend time on it - I'll fix that.

File: CONTRIBUTING.md (hunk 2 above)
Comment (daphne-sfdc) [reply CONTRIBUTING.md:12]: Yes!  Let me add that.

**Pull request merged**